app/components/DocsPopover.tsx

```diff
@@ -45,22 +45,22 @@ export const DocsPopover = ({ heading, icon, summary, links }: DocsPopoverProps)
   return (
     <Popover>
       <PopoverButton className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}>
-        <Info16Icon aria-label="Links to docs" className="shrink-0" />
+        <Info16Icon aria-label="Links to docs" className="shrink-0 text-tertiary" />
```

Comment:
tertiary looks kinda sad next to the brighter refresh button and the stop button. it also looks kinda sad on its own

<img width="307" alt="image" src="https://github.com/user-attachments/assets/6508ed80-7d0d-424f-b84d-efe157ab78cb">

<img width="590" alt="image" src="https://github.com/user-attachments/assets/bae1ebe5-cc5e-4bf9-b001-fda3fab33a6e">

Reply:
Brightened the whole lot up